packages/components/src/components/Alert/Alert.tsx

```diff
@@ -170,13 +167,33 @@ export const Alert: FC<AlertProps> = ({
     </View>
```

Comment:
^^ Looks like we lost `preventScaling` on the alert Icon so it's growing with font changes.

Reply:
Was confused seeing the touchable area not reaching the left edge and the hitSlop math adding up, only when realizing I had my font scaled up that it was due to the icon growing. Should remedy that problem once preventScaling is back.

Reply:
Re-added `preventScaling`. Good catch.